### ISSUE TITLE
Unknown character set: 'utf8mb4_general_ci'" when creating new collection

### ISSUE THREAD

**Issue by LasseRosenow** [directus/directus#11155]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues?q=is%3Aissue) that already describes my problem.

### Describe the Bug

I have a Directus instance on my server. Recently we migrated it to the Directus instance and database server of my customer and now when trying to create a new `files` field, it does not work and we get 3 errors in the following order:

### 1.
```
{
  "errors": [
    {
      "message": "create table `test_collection` (`id` int unsigned not null auto_increment primary key, `status` varchar(255) not null default 'draft', `sort` int null, `user_created` char(36) null, `date_created` timestamp null, `user_updated` char(36) null, `date_updated` timestamp null) default character set utf8mb4_german2_ci - ER_UNKNOWN_CHARACTER_SET: Unknown character set: 'utf8mb4_general_ci'",
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR"
      }
    }
  ]
}
```

### To Reproduce

I am not exactly sure how. I guess it has something to do with the charset?

### What version of Directus are you using?

9.4.3

### What version of Node.js are you using?

16

### What database are you using?

MariaDB

### What browser are you using?

Firefox 95

### What operating system are you using?

Windows

### How are you deploying Directus?

Debian server

**Comment by LasseRosenow** [comment]
It seems to be working now, after we removed the env variable: 

`DB_CHARSET="utf8mb4_general_ci"`

**Comment by stockHuman** [comment]
It's worth noting that the docs suggest an incorrect value; the collation instead of the charset.

![Screen Shot 2022-01-28 at 4 21 25 PM](https://user-images.githubusercontent.com/13852458/151623171-7cfee360-16f5-4f32-9bbe-483fedc2cf93.png)

**Comment by rijkvanzanten** [comment]
@stockHuman That's not a suggested value, that's the actual default value, as per the `mysql` package:

> charset: The charset for the connection. This is called "collation" in the SQL-level of MySQL (like utf8_general_ci). If a SQL-level charset is specified (like utf8mb4) then the default collation for that charset is used. (Default: 'UTF8_GENERAL_CI')

**Comment by stockHuman** [comment]
Yes, I should clarify, _the default value (which is correct) suggests that this option takes a CI string by virtue of (all? most?) other defaults representing examples of valid values_, including the one immediately preceding it.

This seems reinforced by the description text that, while again correct, invites some ambiguity with "Charset/**collation**".

As far as I can tell this is the only required default value in the docs that works that way, where one value (in this case `utf8mb4`) should be input to produce another (`utf8mb4_general_ci`). The property _is_ called `DB_CHARSET`, so hey. Maybe it's worth suggesting a value as well as enumerating the actual default?
